### ISSUE TITLE
Sentence tokenizer fails when sentence.startswith('. ')

### ISSUE THREAD

**Issue by zqwerty** [nltk/nltk#2921]
```
from nltk.tokenize import sent_tokenize
sent_tokenize('. a')
```
will trigger IndexError from `punkt.py:_realign_boundaries`. But `sent_tokenize('. ')` will not.

Envs:
python=3.6
nltk=3.6.6

**Comment by tomaarsen** [comment]
Thank you for the bug report! 
I believe to have resolved the issue in #2922. If you are experiencing that issue, please use the `develop` branch instead:
```
pip install git+https://github.com/nltk/nltk
```

I'll reopen this so others can see this.

**Comment by tomaarsen** [comment]
@zqwerty
NLTK 3.6.7 has been released, which includes the fix for this issue. Thank you for reporting it! I'll close this now.
